tarea_1.py

```diff
@@ -51,10 +51,319 @@
 la tarea.
 
 """
-__author__ = 'escribe_tu_nombre'
+__author__ = 'Erick Fernando López Fimbres'
+
+import entornos_o
+from random import choice
+from random import random
 
-import entorno_o
 
 # Requiere el modulo entornos_o.py
 # Usa el modulo doscuartos_o.py para reutilizar código
 # Agrega los modulos que requieras de python
+
+class SeisCuartos(entornos_o.Entorno):
+    """
+    Clase para un entorno de seis cuartos.
+
+    El estado se define como (robot, A, B, C, D, E, F)
+    donde robot puede tener los valores "A", "B", "C", "D", "E" y "F"
+    A, B, C, D, E, F pueden tener los valores "limpio", "sucio"
+
+    Las acciones válidas en el entorno son ("ir_Izquierda", "ir_Derecha", "limpiar", "nada") en
+    cualquier cuarto ("subir") solo si se encuentra en los cuartos de abajo.
+    ("bajar") solo si se encuentra en los cuartos de arriba ("D","E","F").
+    
+    Los sensores es una tupla (robot, limpio?)
+    con la ubicación del robot y el estado de limpieza
+    """
+    
+    def __init__(self, x0=["A", "sucio", "sucio","sucio", "sucio","sucio", "sucio"]):
+        """
+        Por default inicialmente el robot está en A y losdemas cuartos
+        están sucios
+        """
+        
+        self.x = x0[:]
+        self.desempeño = 0
+
+    def acción_legal(self, acción):
+        acción_legal=True
```

Comment:
Faltaría algo como
`if acción not in ("ir_Izquierda", "ir_Derecha", "limpiar", "nada", "subir", "bajar"): return False`